README.md

```diff
@@ -113,13 +114,23 @@ rails server
 
```

Comment:
- [ ] In your project, these two files are the same tests and they are both model validation tests: `spec\models\category_spec.rb` and `spec\requests\category_spec.rb`. Kindly write the correct tests for the `requests` folder (you can refer to the previous requests project in this module).

- [ ] You have no integration test. Kindly add integration tests.